client-app/shared/layout/components/language-selector/language-selector.vue

```diff
@@ -39,7 +39,7 @@
           lazy
         />
 
-        <span class="language-selector__item-text">
+        <span class="language-selector__item-text capitalize">
```

Comment:
move it to class bellow
&__item-text ...

Reply:
> move it to class bellow &__item-text ...

Fixed